infrastructure/data.tf

```diff
@@ -17,3 +17,7 @@ data "azurerm_monitor_action_group" "common" {
   resource_group_name = var.common_config.resource_group_name
   name                = each.value
 }
+
+data "azurerm_resource_group" "frontdoor" {
+  name = "pins-rg-front-door-${var.environment}-${module.primary_region.location_short}"
+}
```

Comment:
Should we keep front door all in the same resource group as the other resources? Ideally we keep them all grouped and independent.